src/components/panels/index.js

```diff
@@ -429,10 +429,17 @@ const TextPanel = ({ data }) => {
             <section
               key={item + html}
               css={{
+                ':last-of-type': {
+                  '.gatsby-image-wrapper': {
+                    marginBottom: 0
+                  }
+                },
+                ':not(:last-of-type)': {
+                  marginBottom: SPACING.L,
+                  paddingBottom: SPACING.M
+                },
                 borderBottom: item === items.length - 1 ? 'none' : `solid 1px var(--color-neutral-100)`,
```

Comment:
```suggestion
                '& + &': {
                  borderTop: 'solid 1px var(--color-neutral-100)',
                  marginTop: SPACING.M,
                  paddingTop: SPACING.L
                },
```

I just want to show another way to accomplish the same styling. Not a blocker.

Reply:
This is great! Thanks for the more succinct code!